recipes/formidable/README.md

```diff
@@ -0,0 +1,47 @@
+## formidable instance
+
+Now you can send your own `formidable.IncomingForm` instance via `options.IncomingForm`. It means you can do something beyond default behavior, such as handling image upload.
+
+Below is the simple demo of custom `formidable.IncomingForm` instance and event handling. Check [formidable](https://github.com/felixge/node-formidable) for more info.
+
+```js
+'use strict'
+
+var app = require('koa')()
+var path = require('path')
+var IncomingForm = require('formidable')
+var body = require('../../index')
+
+var form = new IncomingForm()
+form.keepExtensions = true
+form.encoding = 'utf-8'
+form.uploadDir = path.join(__dirname, 'uploads')
+form.on('field', function (name, value) {
+  console.log(name, value) // name is user, value is test
+})
+form.on('file', function (name, file) {
+  console.log(name) // => foo
+  console.log(file) // => README.md
+  console.log(file.path) // => full filepath to where is uploaded
+})
+form.on('end', function (name, file) {
+  console.log('finish parse')
+})
+
+app
+  .use(body({
+    IncomingForm: form
+  }))
```

Comment:
add one more middleware to show it continue working as expected? Someone may think that if pass an instance it can't handle `files` and `fields`.